tests/build_provider_packages_dependencies.py

```diff
@@ -114,13 +114,14 @@ class ImportFinder(NodeVisitor):
     AST visitor that collects all imported names in its imports
     """
 
-    def __init__(self, filename):
+    def __init__(self, filename: str) -> None:
+        
```

Comment:
```suggestion
    def __init__(self, filename: str) -> None:
```

(suggested by flake8)